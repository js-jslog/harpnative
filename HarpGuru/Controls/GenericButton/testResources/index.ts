import { ApparatusIds, PozitionIds, PitchIds, getHarpStrata } from 'harpstrata'
import type { HarpStrata, HarpStrataProps, ActiveIds } from 'harpstrata'

const baseHarpStrataProps: HarpStrataProps = {
  apparatusId: ApparatusIds.MajorDiatonic,
  pozitionId: PozitionIds.First,
  harpKeyId: PitchIds.C,
  activeIds: [] as ActiveIds,
}

const firstPozitionHarpStrataProps: HarpStrataProps = {
  ...baseHarpStrataProps,
  pozitionId: PozitionIds.First,
}
const secondPozitionHarpStrataProps: HarpStrataProps = {
  ...baseHarpStrataProps,
  pozitionId: PozitionIds.Second,
}

const keyCHarpStrataProps: HarpStrataProps = {
  ...baseHarpStrataProps,
  harpKeyId: PitchIds.C,
}
const keyDHarpStrataProps: HarpStrataProps = {
  ...baseHarpStrataProps,
  harpKeyId: PitchIds.D,
}

export const firstPozitionHarpStrata: HarpStrata = getHarpStrata(
  firstPozitionHarpStrataProps
)
export const secondPozitionHarpStrata: HarpStrata = getHarpStrata(
  secondPozitionHarpStrataProps
)

export const keyCHarpStrata: HarpStrata = getHarpStrata(keyCHarpStrataProps)
export const keyDHarpStrata: HarpStrata = getHarpStrata(keyDHarpStrataProps)
