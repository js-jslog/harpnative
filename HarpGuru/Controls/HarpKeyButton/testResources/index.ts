import { ApparatusIds, PozitionIds, PitchIds, getHarpStrata } from 'harpstrata'
import type { HarpStrata, HarpStrataProps, ActiveIds } from 'harpstrata'

const baseHarpStrataProps: HarpStrataProps = {
  apparatusId: ApparatusIds.MajorDiatonic,
  pozitionId: PozitionIds.First,
  harpKeyId: PitchIds.C,
  activeIds: [] as ActiveIds,
}
const keyCHarpStrataProps: HarpStrataProps = {
  ...baseHarpStrataProps,
  harpKeyId: PitchIds.C,
}
const keyDHarpStrataProps: HarpStrataProps = {
  ...baseHarpStrataProps,
  harpKeyId: PitchIds.D,
}

export const keyCHarpStrata: HarpStrata = getHarpStrata(keyCHarpStrataProps)
export const keyDHarpStrata: HarpStrata = getHarpStrata(keyDHarpStrataProps)
