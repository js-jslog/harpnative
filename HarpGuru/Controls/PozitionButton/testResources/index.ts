import { ApparatusIds, PozitionIds, PitchIds, getHarpStrata } from 'harpstrata'
import type { HarpStrata, HarpStrataProps, ActiveIds } from 'harpstrata'

const baseHarpStrataProps: HarpStrataProps = {
  apparatusId: ApparatusIds.MajorDiatonic,
  pozitionId: PozitionIds.First,
  harpKeyId: PitchIds.C,
  activeIds: [] as ActiveIds,
}
const firstPozitionHarpStrataProps: HarpStrataProps = {
  ...baseHarpStrataProps,
  pozitionId: PozitionIds.First,
}
const secondPozitionHarpStrataProps: HarpStrataProps = {
  ...baseHarpStrataProps,
  pozitionId: PozitionIds.Second,
}

export const firstPozitionHarpStrata: HarpStrata = getHarpStrata(
  firstPozitionHarpStrataProps
)
export const secondPozitionHarpStrata: HarpStrata = getHarpStrata(
  secondPozitionHarpStrataProps
)
