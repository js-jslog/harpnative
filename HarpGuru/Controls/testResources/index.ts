import { getHarpStrata, ApparatusIds, PozitionIds, PitchIds } from 'harpstrata'
import type { HarpStrataProps } from 'harpstrata'

import type { HarpStrataControlProps } from '../types'

export const baseHarpStrataProps: HarpStrataProps = {
  apparatusId: ApparatusIds.MajorDiatonic,
  pozitionId: PozitionIds.First,
  harpKeyId: PitchIds.C,
  activeIds: [],
}
export const firstPozitionCMajorDiatonicHarpStrataProps: HarpStrataProps = { ...baseHarpStrataProps }
export const secondPozitionCMajorDiatonicHarpStrataProps: HarpStrataProps = { ...baseHarpStrataProps, pozitionId: PozitionIds.Second }

export const firstPozitionCMajorDiatonicHarpStrata = getHarpStrata(firstPozitionCMajorDiatonicHarpStrataProps)
export const secondPozitionCMajorDiatonicHarpStrata = getHarpStrata(secondPozitionCMajorDiatonicHarpStrataProps)

export const exampleHarpStrataControlProps: HarpStrataControlProps = {
  activeHarpStrata: firstPozitionCMajorDiatonicHarpStrata,
  setActiveHarpStrata: jest.fn(),
}
