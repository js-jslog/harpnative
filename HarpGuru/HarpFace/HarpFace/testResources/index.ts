import { ApparatusIds, PozitionIds, PitchIds, getHarpStrata } from 'harpstrata'
import type { HarpStrata, HarpStrataProps, ActiveIds } from 'harpstrata'

import type { HarpFaceProps } from '../index'
import { DisplayModes } from '../index'


const harpStrataProps: HarpStrataProps = {
  apparatusId: ApparatusIds.MajorDiatonic,
  pozitionId: PozitionIds.First,
  harpKeyId: PitchIds.C,
  activeIds: [] as ActiveIds,
}
const activeHarpStrata = getHarpStrata(harpStrataProps)
const { Degree: activeDisplayMode } = DisplayModes
// TODO: If I add a jest.fn() here rather than this nonesense function
// the gradle build fails. I need to figure out to make the testResources
// folders ignored in the build
const setActiveHarpStrata = (activeHarpStrata: HarpStrata): void => {console.log(activeHarpStrata)}

export const exampleHarpFaceProps: HarpFaceProps = {
  activeHarpStrata,
  activeDisplayMode,
  setActiveHarpStrata,
}
