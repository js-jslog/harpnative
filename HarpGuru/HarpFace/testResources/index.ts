import { ApparatusIds, PozitionIds, PitchIds, DegreeIds, getHarpStrata } from 'harpstrata'
import type { HarpStrata, HarpStrataProps, ActiveIds } from 'harpstrata'

import type { HarpFaceProps } from '../HarpFace'
import { DisplayModes } from '../HarpFace'

const allActiveDegrees = [
  DegreeIds.Root, DegreeIds.Flat2, DegreeIds.Second, DegreeIds.Flat3, DegreeIds.Third, DegreeIds.Fourth,
  DegreeIds.Flat5, DegreeIds.Fifth, DegreeIds.Flat6, DegreeIds.Sixth, DegreeIds.Flat7, DegreeIds.Seventh,
]
const { Degree: activeDisplayMode } = DisplayModes
// TODO: If I add a jest.fn() here rather than this nonesense function
// the gradle build fails. I need to figure out to make the testResources
// folders ignored in the build
const setActiveHarpStrata = (activeHarpStrata: HarpStrata): void => {console.log(activeHarpStrata)}

const baseHarpStrataProps: HarpStrataProps = {
  apparatusId: ApparatusIds.MajorDiatonic,
  pozitionId: PozitionIds.First,
  harpKeyId: PitchIds.C,
  activeIds: [] as ActiveIds,
}
const inactiveCellsHarpStrataProps = baseHarpStrataProps
const activeCellsHarpStrataProps = { ...baseHarpStrataProps, activeIds: allActiveDegrees }

const inactiveCellsHarpStrata = getHarpStrata(inactiveCellsHarpStrataProps)
const activeCellsHarpStrata = getHarpStrata(activeCellsHarpStrataProps)


export const inactiveCellsHarpFaceProps: HarpFaceProps = {
  activeHarpStrata: inactiveCellsHarpStrata,
  activeDisplayMode,
  setActiveHarpStrata,
}
export const activeCellsHarpFaceProps: HarpFaceProps = {
  activeHarpStrata: activeCellsHarpStrata,
  activeDisplayMode,
  setActiveHarpStrata,
}
export const harpFaceProps = inactiveCellsHarpFaceProps
