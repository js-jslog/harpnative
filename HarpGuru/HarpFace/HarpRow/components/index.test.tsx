import 'react-native'
import React from 'react'
import { DegreeIds } from 'harpstrata'
import { render } from '@testing-library/react-native'

import { harpFaceProps } from '../../testResources'

import { HarpRow } from './index'

test('The first 3 holes of a blow row from a major diatonic harmonica can be rendered', () => {
  const { getByText } = render(
    <HarpRow {...harpFaceProps} yCoord={2} xRange={[0, 1, 2]} />
  )

  expect(getByText(DegreeIds.Root)).toBeTruthy()
  expect(getByText(DegreeIds.Third)).toBeTruthy()
  expect(getByText(DegreeIds.Fifth)).toBeTruthy()
})

test('A snapshot of a non-home row', () => {
  const { container } = render(
    <HarpRow
      {...harpFaceProps}
      yCoord={0}
      xRange={[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]}
    />
  )

  expect(container).toMatchSnapshot()
})

test('A snapshot of a blow home row', () => {
  const { container } = render(
    <HarpRow
      {...harpFaceProps}
      yCoord={2}
      xRange={[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]}
    />
  )

  expect(container).toMatchSnapshot()
})

test('A snapshot of a draw home row', () => {
  const { container } = render(
    <HarpRow
      {...harpFaceProps}
      yCoord={3}
      xRange={[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]}
    />
  )

  expect(container).toMatchSnapshot()
})
