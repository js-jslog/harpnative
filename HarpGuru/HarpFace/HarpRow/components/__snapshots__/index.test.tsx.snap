// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A snapshot of a blow home row 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#e9ecef",
        "borderBottomLeftRadius": 0,
        "borderBottomRightRadius": 0,
        "borderBottomWidth": 0,
        "borderColor": "#ced4da",
        "borderLeftWidth": 1.1,
        "borderRightWidth": 1.1,
        "borderTopLeftRadius": 12.199000000000002,
        "borderTopRightRadius": 12.199000000000002,
        "borderTopWidth": 1.1,
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "space-around",
      }
    }
  >
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          1
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          3
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          5
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          1
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          3
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          5
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          1
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          3
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          5
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          1
        </Text>
      </View>
    </>
  </View>
</View>
`;

exports[`A snapshot of a draw home row 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#e9ecef",
        "borderBottomLeftRadius": 12.199000000000002,
        "borderBottomRightRadius": 12.199000000000002,
        "borderBottomWidth": 1.1,
        "borderColor": "#ced4da",
        "borderLeftWidth": 1.1,
        "borderRightWidth": 1.1,
        "borderTopLeftRadius": 0,
        "borderTopRightRadius": 0,
        "borderTopWidth": 0,
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "space-around",
      }
    }
  >
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          2
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          5
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          7
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          2
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          4
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          6
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          7
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          2
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          4
        </Text>
      </View>
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          6
        </Text>
      </View>
    </>
  </View>
</View>
`;

exports[`A snapshot of a non-home row 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "transparent",
        "borderBottomLeftRadius": 0,
        "borderBottomRightRadius": 0,
        "borderBottomWidth": 0,
        "borderColor": "#ced4da",
        "borderLeftWidth": 0,
        "borderRightWidth": 0,
        "borderTopLeftRadius": 0,
        "borderTopRightRadius": 0,
        "borderTopWidth": 0,
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "space-around",
      }
    }
  >
    <
      accessible={false}
      disabled={true}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fefefe",
          "borderColor": "#ced4da",
          "borderRadius": 7.5394000000000005,
          "borderWidth": 0,
          "elevation": 0,
          "height": 31.9374,
          "justifyContent": "center",
          "width": 31.9374,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 0,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      />
    </>
    <
      accessible={false}
      disabled={true}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fefefe",
          "borderColor": "#ced4da",
          "borderRadius": 7.5394000000000005,
          "borderWidth": 0,
          "elevation": 0,
          "height": 31.9374,
          "justifyContent": "center",
          "width": 31.9374,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 0,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      />
    </>
    <
      accessible={false}
      disabled={true}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fefefe",
          "borderColor": "#ced4da",
          "borderRadius": 7.5394000000000005,
          "borderWidth": 0,
          "elevation": 0,
          "height": 31.9374,
          "justifyContent": "center",
          "width": 31.9374,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 0,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      />
    </>
    <
      accessible={false}
      disabled={true}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fefefe",
          "borderColor": "#ced4da",
          "borderRadius": 7.5394000000000005,
          "borderWidth": 0,
          "elevation": 0,
          "height": 31.9374,
          "justifyContent": "center",
          "width": 31.9374,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 0,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      />
    </>
    <
      accessible={false}
      disabled={true}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fefefe",
          "borderColor": "#ced4da",
          "borderRadius": 7.5394000000000005,
          "borderWidth": 0,
          "elevation": 0,
          "height": 31.9374,
          "justifyContent": "center",
          "width": 31.9374,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 0,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      />
    </>
    <
      accessible={false}
      disabled={true}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fefefe",
          "borderColor": "#ced4da",
          "borderRadius": 7.5394000000000005,
          "borderWidth": 0,
          "elevation": 0,
          "height": 31.9374,
          "justifyContent": "center",
          "width": 31.9374,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 0,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      />
    </>
    <
      accessible={false}
      disabled={true}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fefefe",
          "borderColor": "#ced4da",
          "borderRadius": 7.5394000000000005,
          "borderWidth": 0,
          "elevation": 0,
          "height": 31.9374,
          "justifyContent": "center",
          "width": 31.9374,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 0,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      />
    </>
    <
      accessible={false}
      disabled={true}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fefefe",
          "borderColor": "#ced4da",
          "borderRadius": 7.5394000000000005,
          "borderWidth": 0,
          "elevation": 0,
          "height": 31.9374,
          "justifyContent": "center",
          "width": 31.9374,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 0,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      />
    </>
    <
      accessible={false}
      disabled={true}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fefefe",
          "borderColor": "#ced4da",
          "borderRadius": 7.5394000000000005,
          "borderWidth": 0,
          "elevation": 0,
          "height": 31.9374,
          "justifyContent": "center",
          "width": 31.9374,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 0,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      />
    </>
    <
      accessibilityRole="button"
      accessible={true}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fefefe",
            "borderColor": "#ced4da",
            "borderRadius": 7.5394000000000005,
            "borderWidth": 1.1,
            "elevation": 0,
            "height": 31.9374,
            "justifyContent": "center",
            "width": 31.9374,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ced4da",
              "fontSize": 12.199000000000002,
            }
          }
        >
          7b
        </Text>
      </View>
    </>
  </View>
</View>
`;
